### PR TITLE
[SPARK-34952][SQL][FOLLOWUP] Normalize pushed down aggregate col name and group by col name

### PR DESCRIPTION
**Pull request by huaxingao**
### What changes were proposed in this pull request?
Normalize pushed down aggregate col names and group by col names ...

### Why are the changes needed?
to handle case sensitive col names

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Modify existing test

### CODE REVIEW

**Comment by huaxingao** [comment]
cc @viirya

File: sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala [hunk 1]
```diff
@@ -93,8 +93,12 @@ object V2ScanRelationPushDown extends Rule[LogicalPlan] with PredicateHelper {
                     agg
                 }
               }
+              aggregates = DataSourceStrategy.normalizeExprs(aggregates, sHolder.relation.output)
```
Comment (viirya) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala:96]: `val normalizedAggregates`? So you can keep `val aggregates` above.

**Review by viirya — APPROVED** [review]
Just a minor comment.

File: sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala (hunk 1 above)
Comment (huaxingao) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala]: Done. Thanks!

**Comment by viirya** [comment]
Pending CI.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Thank you, @huaxingao and @viirya .
Merged to master/3.2.

cc @gengliangwang

**Comment by huaxingao** [comment]
Thanks @dongjoon-hyun @viirya

**Pull request closed without merging**